po/common/fr.po

```diff
@@ -2326,8 +2318,9 @@ msgid "Serving size"
 msgstr "Taille d'une portion"
 
 msgctxt "serving_size_prepared"
-msgid "Prepared serving size"
-msgstr "Taille d'une portion préparée"
+msgid ""
```

Comment:
fr.po has unwanted changes, can we take it out of Crowdin?

Reply:
we fix it in Crowdin. We can't make exceptions for 1 language

Reply:
fixed in new pr